app/components/sidebars/main/channels_list/filtered_list/filtered_list.js

```diff
@@ -396,7 +396,8 @@ class FilteredList extends Component {
                     renderItem={this.renderItem}
                     keyExtractor={(item) => item.id}
                     onViewableItemsChanged={this.updateUnreadIndicators}
-                    keyboardDismissMode='on-drag'
+                    keyboardDismissMode='interactive'
```

Comment:
`keyboardDismissMode='on-drag'` works on android and it provides an alternative to back button.
Do we want to remove that interaction @enahum?

Can we do something like keyboardDismissMore = {Platform.OS === 'android' ? 'on-drag' : 'interactive} instead? so it still keeps the existing behaviour and adds the property only to IOS as it is just an IOS supported property?

Reply:
Fine with me

Reply:
Done.

Reply:
There are 10 other uses of `keyboardDismissMore` in other components, 6 use just `interactive` and the rest use only `on-drag`.

They should probably be made consistent and all use `{Platform.OS === 'ios' ? 'interactive' : 'on-drag'}`, shouldn't they?

Reply:
I think we should look at them on a case by case basis, if it make sense, then yes we make the change

Reply:
@deanwhillier I think we are good here. We can create a new new PR for the other places

Reply:
Feel free to merge it if you agree

Reply:
Sounds good, thanks!